docs/data-sources/service_accounts.md

```diff
@@ -27,10 +27,10 @@ data "twingate_service_accounts" "foo" {
 
 ### Read-Only
 
-- `id` (String) The ID of this resource.
-- `service_accounts` (Block List) List of Service Accounts (see [below for nested schema](#nestedblock--service_accounts))
+- `id` (String) The ID of the Service Accounts datasource.
+- `service_accounts` (Attributes List) List of Service Accounts (see [below for nested schema](#nestedatt--service_accounts))
 
-<a id="nestedblock--service_accounts"></a>
+<a id="nestedatt--service_accounts"></a>
 ### Nested Schema for `service_accounts`
 
 Read-Only:
```

Comment:
For the `id` value we should leave this as "The ID of the Service Account resource" since this is an output, not an input. This is the Terraform ID, correct?